app/api/og/route.tsx

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable @next/next/no-img-element */
+// @ts-nocheck
+import { ImageResponse } from "next/og";
+import { NextRequest } from "next/server";
+import { GoGitPullRequest } from "react-icons/go";
+import { GoCodeReview } from "react-icons/go";
+import { MdUpdate } from "react-icons/md";
+
+export const runtime = "edge";
+
+export async function GET(request: NextRequest) {
+  const url = new URL(request.url);
+  const decodedUrl = decodeURIComponent(url.toString());
+  const replacedUrl = decodedUrl.replace(/&amp;/g, "&");
+  const parsedUrl = new URL(replacedUrl);
+
+  const github_username =
+    url.searchParams.get("github_username") || "Not a user";
+  const content = parsedUrl.searchParams.get("content") || "Contributor";
+  const pr_opened = parsedUrl.searchParams.get("pr_opened") || "0";
+  const pr_reviewed = parsedUrl.searchParams.get("pr_reviewed") || "0";
+  const eod_update = parsedUrl.searchParams.get("eod_update") || "0";
+
+  const logoImageData = await fetch(
+    new URL("../../../public/logo.jpg", import.meta.url),
+  ).then((res) => res.arrayBuffer());
+  const profileImage = `https://avatars.githubusercontent.com/${github_username}`;
+
+  const fontData = await fetch(
+    new URL("../../../public/Roboto-Bold.ttf", import.meta.url),
```

Comment:
contents in public directory can be accessed directly like this:
```suggestion
    new URL("/Roboto-Bold.ttf", import.meta.url),
```

<img width="1600" alt="image" src="https://github.com/coronasafe/leaderboard/assets/25143503/9aee74f2-9839-446d-9ad5-8ddae9bec545">